### DISCUSSION TITLE
Question About Student ID Format for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by mouadhhhallem** [community/community#147789] [GitHub Education]
Hi everyone,  

I’m trying to apply for the GitHub Student Developer Pack, but I’m unsure if my student ID card is acceptable as proof of enrollment. My ID card includes my name on the front, but the year of enrollment (2023/2024) is only on the back. Will this be sufficient, or should I provide additional documentation alongside my ID?  

If anyone has experience with this or knows what GitHub typically accepts, I’d really appreciate your guidance.  

Thank you in advance!

**Comment by davevad93** [comment]
Hi @mouadhhhallem ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student):

- **Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.**

Therefore, if your ID doesn't contain a current date in `DD/MM/YYYY` or similar formats, I suggest that you submit one of the above that meets that requirement , otherwise it wouldn't be sufficient to prove your current student status.
